package.json

```diff
@@ -29,8 +30,10 @@
   "scripts": {
     "dev": "rollup --config --watch",
     "build": "rollup --config && terser dist/camera-controls.js -o dist/camera-controls.min.js --comments '/^!/' && downlevel-dts . .",
+    "prepack": "npm run build",
```

Comment:
This is important: once the [`npm publish`](https://docs.npmjs.com/cli/v9/using-npm/scripts#npm-publish) will be executed, `npm prepack` will build the `dist` folder.

You can try yourself, by:

```sh
$ npm pack
```

Reply:
This way, dist files are build "on demand" just before packing, no need to version them anymore